lib/YuiRestClient/Widget/Label.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YuiRestClient::Widget::Label;
+
+use strict;
+use warnings;
+
+use parent 'YuiRestClient::Widget::Base';
+use YuiRestClient::Action;
```

Comment:
I guess we can remove this import as we don't use anything from that module.

Reply:
Changed.